### ISSUE TITLE
Can't Break Statue

### ISSUE THREAD

**Issue by ItsOnlyValerie** [Tictim/Paraglider#130]
I'm playing on Fabric version 1.20.1. I found a Goddess Statue and wanted to mine it and relocate it, but it disappeared when broken. 
I read through the comments on Curseforge and it looks like a few people commented asking how to relocate it but with no proper answer, I found out about this mod through a Youtube video and the person was playing on 1.19.2, they were able to pick up the Statue and relocate it. 
I'm guessing this is a bug and would like to hear some responses!

**Comment by Tictim** [comment]
Yep, this is a bug. My condolences to the million (gross est.) fabric players with a shitty version of Paraglider that couldn't even pick up statue.

If I saw this sooner, like, a year ago, then I would have tried to fix this, but as of now the development of 20.x versions has stopped in favor of 21.x NeoForge releases. No further releases on 20.x versions are planned, unless a PR for an important fix is made, which will be released in a maintenance release.

**Comment by Tictim** [comment]
Finally fixed as of 20.1.4.
